Dockerfile.dev

```diff
@@ -6,23 +6,39 @@ ENV PYTHONDONTWRITEBYTECODE 1
 # Turns off buffering for easier container logging
 ENV PYTHONUNBUFFERED 1
 
-WORKDIR /app
 
-# Installs Pipenv and extra dependencies
-RUN apt update && apt install pipenv curl zlib1g-dev libssl-dev -y
+# Installs build dependecies
+RUN apt update && apt install -y curl zlib1g-dev libssl-dev build-essential libffi-dev
+
+RUN useradd -ms /bin/bash appuser
+USER appuser
+
+VOLUME ["/app"]
+WORKDIR /app
 
 # Installs Pyenv
-ENV PATH="/root/.pyenv/bin:${PATH}"
+ENV PATH="/home/appuser/.pyenv/bin:/home/appuser/.local/bin:${PATH}"
 RUN curl https://pyenv.run | bash
+RUN export PATH="$(pyenv root)/shims/python:$PATH"
 RUN echo 'eval "$(pyenv init -)"' >> ~/.bashrc
+RUN echo 'eval "$(pyenv init --path)"' >> ~/.bashrc
+# RUN eval "$(pyenv init - sh)"
+# RUN eval "$(pyenv init --path)"
```

Comment:
I assume you only need one of the first to lines, right?
If possible, let's remove the commented lines here please.

Reply:
indeed, thanks!
will remove the commented lines